kaitai-struct.gemspec

```
@@ -25,7 +25,7 @@ EOF
   s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
   s.require_paths = ['lib']
 
-  s.files = `git ls-files`.split("\n")
+  s.files = Dir['lib/**/*.rb'] + Dir['bin/*'] + Dir['spec/*']
```

This repo has only the `lib/kaitai/struct` folder, so it probably makes sense to include just this?

```suggestion
  s.files = Dir['lib/**/*.rb']
```

I'm not even sure whether invoking `Dir['bin/*']` and `Dir['spec/*']` doesn't trigger an exception given that `bin/` and `spec/` folders are not present. Even if it doesn't, mentioning these non-existent folders is at least confusing.